### ISSUE TITLE
Text area when cleaned the label stays up #2.0-preview

### ISSUE THREAD

**Issue by bmodeprogrammer** [PolymerElements/paper-input#508]
### Description
When set value of textarea to `''` the label continue up;

### Expected outcome
Label come down again
### Actual outcome

the label stays up

### Live Demo
http://plnkr.co/edit/TdFUkUpTH1lwn8kCJw75?p=preview

### Steps to reproduce

```
<iron-form>
  <form>
  <paper-textarea name="name" value="Test" label="Name" required></paper-textarea>
  <br>
  <paper-button onclick="document.querySelector('paper-textarea').value = ''" raised>Clean</paper-button>
</form>
</iron-form>
```
I only tested with chrome!
### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by digint** [comment]
Note that while the pull request: PolymerElements/iron-autogrow-textarea#106 fixes this issue, setting the value to `undefined` still produces the same error (because of: fccce510f7c69719029782842690040825bd1e12)

**Comment by bmodeprogrammer** [comment]
@digint yes, this works, I tested. Thank you!

**Comment by lpellegr** [comment]
@notwaldorf Do you know an existing workaround or do you have any update regarding this issue?

**Comment by stramel** [comment]
Dupe #555

**Comment by notwaldorf** [comment]
Looking at the PR right now! Hold tight :)

**Comment by notwaldorf** [comment]
(fixed with the 2.1.1 release of iron-autogrow-textarea btw)
